nixos/doc/manual/installation/installing.xml

```diff
@@ -279,7 +279,8 @@ Retype new UNIX password: ***
 
   </listitem>
 
-  <listitem><para>If everything went well:
+  <listitem><para>To prevent the password prompt, set `users.mutableUsers = false;` in `configuration.nix`, 
+              which allows unattended installation necessary in automation. If everything went well:
```

Comment:
Maybe:

```xml
<listitem><note><para>To prevent the password prompt, set <code>users.mutableUsers = false;</code> in <filename>configuration.nix</filename>, which allows unattended installation necessary in automation.</para></note><para>If everything went well:
```